content/administration/maintain/neutralized_databases.rst

```diff
@@ -0,0 +1,36 @@
+=====================
+Neutralized databases
+=====================
+
+What is a neutralized database?
+===============================
+
+A neutralized database is a non-production database on which several parameters are deactivated. This enables one to
+carry out tests without the risk of launching specific automated processes that could impact production data (e.g.,
+sending emails to customers). Live access is then removed and turned into a testing environment.
+
+.. note::
+   **Any testing database created is a neutralized database:**
+   - testing backup databases
+   - duplicate databases
+   - For Odoo.sh: staging and development databases
+
+.. warning::
+   A database can also be neutralized when upgrading, as it is vital to do some tests before switching to a new version.
+
+The consequences of a neutralized database
+==========================================
+
+Here is a non-exhaustive list of the deactivated parameters:
+- all planned actions (where many flows are generated, e.g., automatic invoicing of subscriptions, mass mailing, etc.)
```

Comment:
```suggestion
- all planned actions (e.g., automatic invoicing of subscriptions, mass mailing, etc.)
```
The first part of the explanation in brackets isn't very clear, plus it doesn't seem necessary.